test/integration/test_s3.py

```diff
@@ -165,7 +165,7 @@ def test_s3_backup():
 	do_s3_backup(0)
 
 def test_s3_backup_multiple_files():
```

Comment:
Any fail test? Maybe that is not possible? Just looking for confirmation it is working as intended

Reply:
Adding new tests to these isn't easy but negative tests are something I'll look into.